recipes/unmerge/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set name = "unmerge" %}
+{% set version = "1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/andvides/unmerge/releases/download/v1.0/unmerge_v1.zip
+  sha256: 9e94ddae669762523c5873e68fb3b6d5d6ac9c8b4d4b7d19aed91cb31275d4cc
+  
+build:
+  number: 0
+  skip: True  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    
+test:
+  commands:
+    - unmerge -h 2>&1 | grep "unmerge" > /dev/null
```

Comment:
Please change the name of this tool. Super generic names like "unmerge" are going to clash with other things. BTW, I think both seqtk and bbtools have options to do this.